src/legacy/status_im/browser/core.cljs

```diff
@@ -315,7 +315,9 @@
                   (navigation/change-tab :browser-stack)
                   (update-browser browser)
                   (resolve-url nil))
-        (.openURL ^js react/linking (url/normalize-url url))))))
+        (do
+          (.openURL ^js react/linking (url/normalize-url url))
+          nil)))))
```

Comment:
I wonder do we need a comment or even to put `nil` in an explanatory def 🤔

Reply:
Returning an map/nil is part of the declaration of events. I don't think it's needed 🤔 But let me know if you want me to add it

Reply:
Makes sense. Np for me either way, just wanted to raise the question. 👍

Reply:
`openURL` is a function with side-effects and shouldn't be part of the event handler. There should be a re-frame effect wrapping this, so that we could do:

```clojure
{:fx [...
      [:linking/open-url (url/normalize-url url)]]}
```

Probably better would be to also put the `normalize-url` call inside the effect since it's probably always desirable as far as I checked the code now. With the effect we won't have to worry about returning nil.

The new effect could be like this:

```clojure
{:fx [...
      [:linking/open-url [url {:normalize? true}]]
      ...]}
```

Reply:
@ilmotta Done. Please check again

Reply:
Nice @mmilad75, LGTM, thanks!